modules/datadog-agent/default.auto.tfvars

```diff
@@ -12,9 +12,7 @@ repository = "https://helm.datadoghq.com"
 
 chart = "datadog"
 
-# https://github.com/DataDog/helm-charts/releases
-# 2.26.2 released on Nov 19, 2021
```

Comment:
If we keep this, then we can quickly see what the version is. Otherwise, you gotta google it, no ?

Reply:
I was thinking we remove that. Googling it can be a pain, but how often is that going to be the case when we can have a bot update this

Reply:
ya but when will that happen